.github/workflows/ci.yml

```diff
@@ -84,28 +84,31 @@ jobs:
         name: Prepare LD_LIBRARY_PATH (Ubuntu only)
         run: echo LD_LIBRARY_PATH=${pythonLocation}/lib >> $GITHUB_ENV
 
-      - name: Build docs
-        run: cargo doc --features "num-bigint num-complex hashbrown" --verbose --target ${{ matrix.platform.rust-target }}
+      # Validate docs have no warnings (except on PyPy; auto-initialize feature fails the compile until `cfg(rustdoc)` is stable).
+      - if: matrix.python-version != 'pypy-3.6'
+        name: Build docs
+        run: cargo doc --no-default-features --features "macros num-bigint num-complex hashbrown" --verbose --target ${{ matrix.platform.rust-target }}
 
-      - name: Build without default features
+      - name: Build (no features)
         run: cargo build --no-default-features --verbose --target ${{ matrix.platform.rust-target }}
```

Comment:
It looks like that `auto-initialize` and `macros` are enabled in this build, though I don't know why 😕

Reply:
It's because of the `Cargo.toml` changes - see other comment.